DFC.App.DiscoverSkillsCareers/ViewModels/Assessment/AssessmentEmailPostResponse.cs

```diff
@@ -6,7 +6,10 @@ namespace DFC.App.DiscoverSkillsCareers.ViewModels
     [ExcludeFromCodeCoverage]
     public class AssessmentEmailPostResponse
     {
-        [Required]
+        [Required(ErrorMessage = "Enter an email address")]
+        [RegularExpression(
```

Comment:
Did you get the regex from here? -> https://skillsfundingagency.atlassian.net/wiki/spaces/DFC/pages/1919091155/CUI+field+validation+rules